inc/com/centreon/engine/notification.hh

```diff
@@ -30,7 +30,7 @@ class contact;
 class notification {
  public:
   notification(notifier::reason_type type, std::string const& author, std::string const& message, uint32_t options, uint64_t notification_id);
-  int execute(std::list<std::shared_ptr<contact>> const& to_notify);
+  int execute(std::unordered_set<contact*> const& to_notify);
```

Comment:
if you want there is already a type name set_pair_list which look like this one. We can perhaps update this type to switch from set to unordered_set...

Reply:
Did not found it. You will probably do it!